### ISSUE TITLE
copy pasting acts weird

### ISSUE THREAD

**Issue by olivermt** [KronicDeth/intellij-elixir#239]
Lets say we have this function:

```
  def generate_recovery(conn, %{"username" => username}) do
    #indempotent call
    UserService.generate_recovery(username)
    conn
    |> json(%{ data: %{mail: username}})
  end````

If I take this part:
```
    conn
    |> json(%{ data: %{mail: username}})```

and copy paste it on the line below, with the caret at the same indentation level it ends up as:
```
  def generate_recovery(conn, %{"username" => username}) do
    #indempotent call
    UserService.generate_recovery(username)
    conn
    |> json(%{ data: %{mail: username}})
        conn
        |> json(%{ data: %{mail: username}})
  end
```

As if the editor does not recognize the code style or something similar.

I saw that #98 is supposedly done, but if I do `def foo() do <enter>` I dont end up two spaces in, but on the same level as the def.

Am I looking for something that has not been finished or is there some config I am missing?

**Comment by KronicDeth** [comment]
I'm not sure why you would think #98 is done.  If it was done the issue would be marked Closed with a merged PR not Open as it is now.  It's still open, so this is a duplicate of that and not a bug.  The feature does not exist yet.

**Comment by olivermt** [comment]
Haha, I read "closed" in the issue, but that was issue #91!

No wonder it doesnt work.
Back to waiting :)
